fplus-lib/src/external_services/github.rs

```diff
@@ -156,12 +151,25 @@ impl GithubWrapper {
             }))
             .build()
             .expect("Could not create Octocrab instance");
-        let iod: InstallationId = installation_id.into();
-        let installation = octocrab.installation(iod);
-        Self {
-            owner,
-            repo,
-            inner: Arc::new(installation),
+        if installation_id.is_some() {
+            let pared_istallation_id: u64 = installation_id
+                .expect("Installation Id exist")
+                .try_into()
+                .expect("Installation Id sucessfully parsed to u64");
+            let iod: InstallationId = pared_istallation_id.into();
+
+            let installation = octocrab.installation(iod);
+            Self {
+                owner,
+                repo,
+                inner: Arc::new(installation),
+            }
+        } else {
+            Self {
+                owner,
+                repo,
+                inner: Arc::new(octocrab),
+            }
```

Comment:
Make the `if` return just `octocrab`/`installation` and use just a single `Self { ... }` after the `if`